views/json/page/default.php

```diff
@@ -11,10 +11,3 @@
 elgg_set_http_header("Content-Type: application/json;charset=utf-8");
 
 echo $vars['body'];
-
-// backward compatibility
-global $jsonexport;
```

Comment:
It's deprecated without a workaround. We had to add it back because it broke webservices.